app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def name_supermarket
+    supermarket.name
+  end
 end
```

Comment:
For user story 3, this is where you can add a model method to count the number of customers an Item has. It could look something like this (since there is an association set up with customers, you can call `.customers` directly
```
def customer_count
    self.customers.count
```

Since we don't want `.count` in the view, we instead call this model method from the view to list the count of customers on the items index page